src/python/WMCore/WMBS/MySQL/Subscriptions/CountFinishedSubscriptionsByWorkflow.py

```diff
@@ -0,0 +1,42 @@
+"""
+_CountFinishedSubscriptionsByWorkflow_
+
+MySQL implementation of Subscription.CountFinishedSubscriptionsByWorkflow
+"""
+
+from WMCore.Database.DBFormatter import DBFormatter
+
+
+class CountFinishedSubscriptionsByWorkflow(DBFormatter):
+    """
+    Gets count subscriptions given workflow but exculding cleanup and log collect subscription
+    """
+
+    sql = """SELECT ww.name as workflow,
+             COUNT(case when ws.finished = 1 then 1 else null end) as finished,
+             COUNT(case when ws.finished = 0 then 1 else null end) as open
```

Comment:
how about we fetch only "open" subscriptions? In the end we don't really care about finished ones and it might buy us 1ns :-)

Reply:
BTW, did you try this query in an oracle backend?

Reply:
I think performance difference are negligible. and this can be used (with high priority). 😄 
Yes I ran this in Oracle.